src/components/Tabs/TabList/__tests__/__snapshots__/tabList-snapshot-tests.jest.js.snap

```diff
@@ -2,11 +2,12 @@
 
 exports[`TabList renders correctly stretched tab list 1`] = `
 <div
-  className="tabs--wrapper stretched"
+  className="tabsWrapper tabs--wrapper stretched"
+  data-testid="tab-list"
   id=""
 >
   <ul
-    className="tabs-list"
+    className="tabsList tabs-list undefined"
```

Comment:
Is this undefined is because of jest&css module integration issues or is something is not working? What happen in prod?

Reply:
It was due to `size` prop, which is undefined by default. I've fixed it